### DISCUSSION TITLE
Getting -7 LIBUSB_ERROR_TIMEOUT

### DISCUSSION THREAD

**Discussion by robin75411** [libusb/libusb#1343] [Q&A]
Requesting support for libusb_error_timeout for Identiv reader connection with an ubuntu machine.
libusb version - 1.0.0
libccid -1.4.31
----------------

I did some more debugging and found out that communication via python script with reader gets stuck  at this call for 110 seconds.

connection.transmit(READ_NDEF_DATA) ant returns with a timeout in  pcscd[10529]: 99999999 ccid_usb.c:905:ReadUSB() read failed (1/2): -7 LIBUSB_ERROR_TIMEOUT

Would you be able to throw some light into this issue and potential root cause and fix? Some of the suggestions we are receiving are around changing
the cabling to active repeater cable for reader and ubuntu machine connectivity. Also, is there a way to override the timeout to say 2/3 seconds.

**Comment by LudovicRousseau** [comment]
This is a duplicate of https://github.com/libusb/libusb/issues/1344
